### ISSUE TITLE
Header / Footer shared across multiple applications / sites

### ISSUE THREAD

**Issue by stevepessah** [akcdotorg/akc.org#12]
Currently the Header and Footer navs have to be manually kept in sync whenever a change is made in AKC.org. I would like to convert these into javascript widgets with self-contained javascript/CSS and absolute URLs and configurable domain names. Also the implementation should be performance-conscience as well cacheable in the consumer site so that one site is not dependent on the other on a real-time basis.

In short: we want to make a 'wrapper' file, which can be served by www.akc.org, and be consumed by other sites. It should contain the header and footer, and have all relevant styles and images served from a common place. We'll probably want to carefully namespace our styles and scripts – so that the content in the content-well of the wrapper doesn't pollute the outer wrapper.

**Comment by kylejbertelsen** [comment]
See #168. This should be a parent ticket for that.

**Comment by stevepessah** [comment]
Moved to JIRA
